astropy/visualization/mpl_normalize.py

```diff
@@ -26,7 +26,7 @@
 )
 
 try:
-    import matplotlib  # noqa: F401
+    import matplotlib as mpl  # noqa: F401
```

Comment:
```suggestion
```
Just tested this with and without matplotlib installed, and it seems this can indeed go for good.